applications/style.css

```diff
@@ -157,7 +158,7 @@ body {
 .application-card {
   border-radius: 15px;
   box-shadow: var(--elevation-1);
-  padding: 15px;
+  padding: 24px;
```

Comment:
NIT : would be better if we use rem instead of px

Reply:
Initially I has used rem here. 
But then I thought of these questions
1. Do we need `rem` for padding? 
2. Do we need to adapt to user preferences for the padding?

Acc. to my opinion, we don't need rem here